### PR TITLE
Update the EAP authentication documentation (with local account)

### PR DESCRIPTION
**Pull request by lzammit**
# Description
Update the documentation for the EAP authentication using local account.

# Impacts
Documentation, administration guide.

# Delete branch after merge
YES

### CODE REVIEW

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 1]
```diff
@@ -1231,34 +1231,20 @@ NOTE: For this feature to work, the users' passwords must be stored in cleartext
...
+# Uncomment the following line to enable local PEAP authentication
+packetfence-local-auth
 ----
 
-CAUTION: You will need to deasactivate password hashing in the database for local authentication to work. In the administration interface, go in 'Configuration -> Advanced' and set 'Database passwords hashing method' to `plaintext`
+Restart the radiusd service in order to apply the change.
+
+/usr/local/pf/bin/pfcmd service radiusd restart
+
+CAUTION: You will need to deasactivate password hashing in the database for local authentication to work. In the administration interface, go in 'Configuration -> Advanced' and set 'Database passwords hashing method' to `plaintext` or `ntlm`. Don't use `bcrypt`.
```
Comment (julsemaan) [inline docs/PacketFence_Administration_Guide.asciidoc]: desactivate

Comment (jrouzierinverse) [reply docs/PacketFence_Administration_Guide.asciidoc]: deactivate

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 2]
```diff
@@ -1244,7 +1244,7 @@ Restart the radiusd service in order to apply the change.
 
 /usr/local/pf/bin/pfcmd service radiusd restart
 
-CAUTION: You will need to deasactivate password hashing in the database for local authentication to work. In the administration interface, go in 'Configuration -> Advanced' and set 'Database passwords hashing method' to `plaintext` or `ntlm`. Don't use `bcrypt`.
+CAUTION: You will need to desactivate password hashing in the database for local authentication to work. In the administration interface, go in 'Configuration -> Advanced' and set 'Database passwords hashing method' to `plaintext` or `ntlm`. Don't use `bcrypt`.
```
Comment (jrouzierinverse) [inline docs/PacketFence_Administration_Guide.asciidoc]: deactivate

Comment (extrafu) [reply docs/PacketFence_Administration_Guide.asciidoc]: Shouldn't we use "disable" ? I think it's better and more common Zenglish than deactivate.

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 3]
```diff
@@ -1231,34 +1231,20 @@ NOTE: For this feature to work, the users' passwords must be stored in cleartext
 Option 5: EAP Local user Authentication
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The goal here is to use the local user you created in the admin GUI for EAP authentication.
-The logic is exactly the same than in option 4, the difference is that we use another SSID and we only use local accounts.
+The goal here is to use the local user to authenticate 802.1X device.
```
Comment (fdurand) [inline docs/PacketFence_Administration_Guide.asciidoc:1234]: use local PacketFence's users to ...

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 4]
```diff
@@ -1231,34 +1231,20 @@ NOTE: For this feature to work, the users' passwords must be stored in cleartext
...
+# Uncomment the following line to enable local PEAP authentication
+packetfence-local-auth
 ----
 
-CAUTION: You will need to deasactivate password hashing in the database for local authentication to work. In the administration interface, go in 'Configuration -> Advanced' and set 'Database passwords hashing method' to `plaintext`
+Restart the radiusd service in order to apply the change.
+
+/usr/local/pf/bin/pfcmd service radiusd restart
+
+CAUTION: You will need to disable password hashing in the database for local authentication to work. In the administration interface, go in 'Configuration -> Advanced' and set 'Database passwords hashing method' to `plaintext` or `ntlm`. Don't use `bcrypt`.
```
Comment (fdurand) [inline docs/PacketFence_Administration_Guide.asciidoc:1247]: isn't it nthash ?

**Comment by lzammit** [comment]
capture d ecran 2017-01-12 a 08 56 37

File: docs/PacketFence_Administration_Guide.asciidoc (hunk 4 above)
Comment (lzammit) [reply docs/PacketFence_Administration_Guide.asciidoc:1247]: capture d ecran 2017-01-12 a 08 56 37

Comment (lzammit) [reply docs/PacketFence_Administration_Guide.asciidoc:1247]: In the french version

Comment (julsemaan) [reply docs/PacketFence_Administration_Guide.asciidoc:1247]: Who uses the french version besides you and Fabrice ?

Comment (lzammit) [reply docs/PacketFence_Administration_Guide.asciidoc:1247]: French users !

File: docs/PacketFence_Administration_Guide.asciidoc (hunk 2 above)
Comment (tlaurion) [reply docs/PacketFence_Administration_Guide.asciidoc]: disable is better.

**Pull request merged**